.circleci/config.yml

```diff
@@ -169,13 +169,43 @@ jobs:
       - checkout
       - restore_cache:
           key: dependency-cache-{{ checksum "package-lock.json" }}
+      - run:
+          name: Set up NVM
+          command: |
+            echo ${NVM_DIR}
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install v10.15.3
+            nvm alias default v10.15.3
+      - run:
+          name: install modules
+          command: |
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            node --version
+            npm install
       - run:
           name: Create dir for test results
-          command: mkdir -p ./test/results
+          command: | 
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            mkdir -p ./test/results
+      - run:
+          name: Build docker images
+          command: |
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            docker-compose up -d 
+            chmod +x ./scripts/wait4.js
+            sh -c "./scripts/wait4.js my-sql"
```

Comment:
that is a great idea to use wait4.js here, but you should wait on `auth-service` not only on `my-sql` so you need to extend by adding new service `auth-service` to wait4.config.js and use `depends` list with `my-sql` on it

Reply:
In actuality, we don't need the auth-service container at all since all the integration tests are handled externally by jest. We just need a mysql container, then execute `npm run test:integration`  and the jest runner will deal with everything. In reality, I should just run `docker-compose up -d mysql` instead of just `docker-compose up -d` and just get a working sql container up.